packages/core/core/src/requests/DevDepRequest.js

```diff
@@ -190,14 +190,22 @@ export async function runDevDepRequest<TResult>(
   await api.runRequest<null, DevDepRequestResult | void>({
     id: 'dev_dep_request:' + devDepRequest.specifier + ':' + devDepRequest.hash,
     type: requestTypes.dev_dep_request,
+    incompleteRequest: !(
+      devDepRequest.invalidateOnFileChange &&
+      devDepRequest.invalidateOnFileCreate
+    ),
     run: ({api}) => {
-      for (let filePath of nullthrows(devDepRequest.invalidateOnFileChange)) {
+      for (let filePath of nullthrows(
+        devDepRequest.invalidateOnFileChange,
+        'DevDepRequest missing invalidateOnFileChange',
+      )) {
         api.invalidateOnFileUpdate(filePath);
         api.invalidateOnFileDelete(filePath);
       }
 
       for (let invalidation of nullthrows(
```

Comment:
We should probably look at adding a lint rule to ensure `nullthrows` always has a description

Reply:
Could just change the type definition. It'd be a bit of work though 😅